wp-offline-shell/readme.txt

```diff
@@ -8,11 +8,11 @@ Stable tag: 0.3.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
-A WordPress plugin for caching theme assets via a service worker for the sake of performance and offline functionality.
+A WordPress plugin for caching theme assets via the [ServiceWorker API](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers) to improve performance and provide offline functionality.
```

Comment:
It's a bit of a stretch though, some people could be mislead to think it actually makes the website work offline.